.changeset/purple-countries-think.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components': patch
+---
+
+Add value field to textArea
```

Comment:
maybe this could be slightly reworded to mention that it passes through the value field as expected so the component works as a controlled component . As the value field was already added on the component level, just not wired up internally to the inner `textarea`

Reply:
I'd thought the lower case `t` indicated that but happy to change